tests/sys_arduino/arduino-test.sketch

```diff
@@ -0,0 +1,100 @@
+/*
+ * Copyright (C) 2018 Federico Pellegrin <fede@evolware.org>
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup     tests
+ * @{
+ *
+ * @file
+ * @brief       Test application for arduino module
+ *
+ * @author      Federico Pellegrin <fede@evolware.org>
+ *
+ * @}
+ */
+
+#include <stdlib.h>
+#include <string.h>
+
+#ifdef STDIO_UART_BAUDRATE
+#define SERIAL_BAUDRATE STDIO_UART_BAUDRATE
+#else
+#define SERIAL_BAUDRATE 115200
+#endif
+
+/* Input buffer for receiving chars on the serial port */
+char buf[64];
+
+/* Counter of received chars in currently receiving line */
+int count = 0;
+
+void setup(void)
+{
+    Serial.begin(SERIAL_BAUDRATE);
+    Serial.println("Hello Arduino!");
+}
+
+void loop(void)
+{
+    /* Read chars if available and seek for CR or LF */
+    while (Serial.available() > 0) {
+        int tmp = Serial.read();
+        if (tmp == '\n' || tmp == '\r') {
+           buf[count] = 0;
+
+           if (count > 1) {
+                if (strncmp(buf, "echo", 4) == 0) {
```

Comment:
You are not testing this command in the Python script.

Reply:
Thanks for checking out and for spotting I forgot the first and most basic test I had in mind to insert :) 

I added now a test to use that echoing command and squashed up the changes.